ecl/hql/hqlexpr.cpp

```diff
@@ -6009,7 +6011,7 @@ void CHqlSelectBaseExpression::setOperands(IHqlExpression * left, IHqlExpression
 {
     //Need to be very careful about the order that this is done in, since queryType() depends on operand2
     unsigned max = attr ? 3 : 2;
-    operands.ensure(max);
+    operands.ensureSpace(max);
```

Comment:
shouldn't this be a ensureCapacity(max) ?

Reply:
They will have an identical effect, it doesn't really matter..